src/tim/vx/tensor_private.h

```diff
@@ -50,6 +50,7 @@ class TensorImpl : public Tensor {
   bool IsConstTensor() {
     return spec_.attr_ == tim::vx::TensorAttribute::CONSTANT;
   }
+  const void* GetDataRef() const { return data_; }
```

Comment:
what if this is not ConstTensor?

Reply:
When re-construct the constant tensor, the API is called only. So there is no impact if it's not constant tensor.